packages/server/src/api/routes/tests/viewV2.spec.ts

```diff
@@ -170,17 +247,33 @@ describe.each([
         }
         const res = await config.api.viewV2.create(newView)
 
-        expect(res).toEqual({
+        const expected: ViewV2 = {
           ...newView,
           schema: {
             id: { visible: true },
             Price: {
               visible: true,
             },
           },
+          queryUI: {
+            groups: [
+              {
+                logicalOperator: UILogicalOperator.ALL,
```

Comment:
There's a part of me wonder if this might be a bit more visually obvious if we had something in the structure utilities to create a default query UI, I assume that is what all these additions are for, to create the kind of basic view that the frontend will send up.

Reply:
I do have a helper further down in the file called `simpleQuery`. For now I've removed the optional fields (they weren't optional when I started) to reduce the verbosity.